src/Telegram.Bot/Requests/RequestBase.cs

```diff
@@ -17,6 +22,10 @@ public abstract class RequestBase<TResponse> : IRequest<TResponse>
         /// <inheritdoc />
         public string MethodName { get; protected set; }
 
+#if NETSTANDARD2_0
+        private static readonly JsonSerializerOptions _serializerOptions = new JsonSerializerOptions { PropertyNamingPolicy = JsonNamingPolicy.CamelCase };
```

Comment:
I used a combination of default snake case naming policy in serialization of single word properties (`Photo` property to `photo` JSON field) and `JsonPropertyName` attribute for multiple words (`FirstName` property to `first_name` JSON field)

Reply:
That's doable, but quite laborious to mark all the needed props. But default value types is still a problem that I don't see a solution for without breaking compatibility.

And I don't really like the idea that webhook users will need to set custom `JsonSerializerOptions` specifically for Bot API, because these options are global for the entire middleware pipeline in ASP.NET Core. If I'd want to host telegram webhook in the same process as my web app options needed for the library might not be compatible with the app. With the current setup we don't even need custom serializer options.